ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -302,3 +309,26 @@ def stop(self):
         """
         self.metadata.close()
         self.processor.close()
+
+    def _retrieve_service_connection_if_needed(self):
+        # We override the current serviceConnection source object if source workflow service already exists in OM.
```

Comment:
Sorry for being picky 😬 can we turn this whole comment block into a dockstring? after that all good